### PR TITLE
style: snapshot theme

### PR DESCRIPTION
**Pull request by lelenei**
## Description

Closes mf-1561

## Type of change

- [ ] Documentation
- [x] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/mask/src/plugins/Snapshot/SNSAdaptor/LoadingFailCard.tsx [hunk 1]
```diff
@@ -33,19 +33,18 @@ export class LoadingFailCard extends Component<
                             marginBottom: 4,
                             marginTop: 32,
                         }}
+                        variant="roundedContained"
                         onClick={() => {
                             this.setState({ error: null })
                             this.props.retry()
                         }}>
-                        Retry
+                        Reload
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Snapshot/SNSAdaptor/LoadingFailCard.tsx]: i18n

File: packages/mask/src/plugins/Snapshot/SNSAdaptor/LoadingFailCard.tsx [hunk 2]
```diff
@@ -33,19 +33,18 @@ export class LoadingFailCard extends Component<
...
                     </Button>
                 </Box>
             ) : (
                 <SnapshotCard title={this.props.title}>
                     <Box style={{ textAlign: 'center' }}>
-                        <Typography color="textPrimary">
-                            Loading fails due to Snapshot API service breakdown.
-                        </Typography>
+                        <Typography color="#07101b">Loading fails due to Snapshot API service breakdown.</Typography>
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Snapshot/SNSAdaptor/LoadingFailCard.tsx]: i18n

File: packages/mask/src/plugins/Snapshot/SNSAdaptor/LoadingFailCard.tsx [hunk 3]
```diff
@@ -20,7 +20,7 @@ export class LoadingFailCard extends Component<
             return this.props.isFullPluginDown ? (
                 <Box style={{ textAlign: 'center', padding: 16 }}>
                     <Typography textAlign="center" color="error">
-                        Loading fails due to Snapshot API service breakdown.
+                        Load failed
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Snapshot/SNSAdaptor/LoadingFailCard.tsx]: i18n

File: packages/mask/src/plugins/Snapshot/SNSAdaptor/ResultCard.tsx [hunk 4]
```diff
@@ -72,6 +77,7 @@ function Content() {
     const {
         payload: { results },
     } = useResults(identifier)
+    console.log('result', results)
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Snapshot/SNSAdaptor/ResultCard.tsx]:
```suggestion
```

File: packages/mask/src/plugins/Snapshot/SNSAdaptor/LoadingCard.tsx [hunk 5]
```diff
@@ -29,6 +29,7 @@ export function LoadingCard(
                         <Skeleton
                             key={i}
                             className={classes.skeleton}
+                            sx={{ backgroundColor: 'gray' }}
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Snapshot/SNSAdaptor/LoadingCard.tsx]: Use a theme color stead.

File: packages/mask/src/plugins/Snapshot/SNSAdaptor/Snapshot.tsx [hunk 6]
```diff
@@ -17,55 +20,55 @@ const useStyles = makeStyles()((theme) => {
...
+        content: {
             flex: 1,
-            maxHeight: 'calc(var(--contentHeight) - var(--tabHeight))',
             overflow: 'auto',
+            borderRadius: '0 0 12px 12px',
             scrollbarWidth: 'none',
             '&::-webkit-scrollbar': {
                 display: 'none',
             },
+            background: '#fff !important',
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Snapshot/SNSAdaptor/Snapshot.tsx]: What about dark mode? Please use a theme color variable.

File: packages/mask/src/plugins/Snapshot/SNSAdaptor/Snapshot.tsx [hunk 7]
```diff
@@ -17,55 +20,55 @@ const useStyles = makeStyles()((theme) => {
...
+            color: theme.palette.maskColor.second,
+            '&:hover': {
+                background: 'transparent',
+            },
+        },
+        tabActive: {
+            background: '#fff',
+            color: theme.palette.maskColor.publicMain,
+            '&:hover': {
+                background: '#fff',
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Snapshot/SNSAdaptor/Snapshot.tsx]: What about dark mode? Please use a color variable stead.

File: packages/mask/src/plugins/Snapshot/SNSAdaptor/Snapshot.tsx [hunk 8]
```diff
@@ -75,64 +78,77 @@ export function Snapshot() {
...
+            <Box className={classes.header}>
+                <Avatar src={resolveIPFS_URL(proposal.space.avatar)} className={classes.avatar} />
+                <Box className={classes.title}>
+                    <Box sx={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center' }}>
+                        <Typography fontSize={18} fontWeight="bold">
+                            {proposal.space.name}
+                        </Typography>
+                        <Box sx={{ display: 'flex' }}>
+                            <Typography fontSize={14} sx={{ paddingRight: 1 }}>
+                                by
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Snapshot/SNSAdaptor/Snapshot.tsx:113]: i18n

File: packages/mask/src/plugins/Snapshot/SNSAdaptor/SnapshotCard.tsx [hunk 9]
```diff
@@ -14,9 +14,10 @@ const useStyles = makeStyles()((theme) => {
             '&:last-child': {
                 marginBottom: 0,
             },
+            background: '#fff',
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Snapshot/SNSAdaptor/SnapshotCard.tsx]: What about dark mode?

File: packages/mask/src/plugins/Snapshot/SNSAdaptor/SnapshotTab.tsx [hunk 10]
```diff
@@ -4,8 +4,11 @@ const useStyles = makeStyles()((theme) => {
     return {
         root: {
             width: '100%',
-            borderTopLeftRadius: 0,
-            borderTopRightRadius: 0,
+            borderRadius: 0,
+            backgroundColor: '#fff !important',
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Snapshot/SNSAdaptor/SnapshotTab.tsx]: Be caution with dark mode.

File: packages/mask/src/plugins/Snapshot/SNSAdaptor/VotingCard.tsx [hunk 11]
```diff
@@ -20,24 +20,21 @@ import { getSnapshotVoteType } from '../utils.js'
...
-            },
-        },
+        choiceButton: {},
         buttonActive: {
-            border: '1px solid rgb(29, 161, 242)',
-            boxShadow: 'inset 0 0 2px rgb(29, 161, 242)',
-            backgroundColor: 'transparent',
-            color: theme.palette.mode === 'dark' ? 'white' : 'black',
+            backgroundColor: theme.palette.maskColor.dark,
+            color: '#fff',
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Snapshot/SNSAdaptor/VotingCard.tsx]: Use a theme variable.

File: packages/mask/src/plugins/Snapshot/SNSAdaptor/Snapshot.tsx [hunk 12]
```diff
@@ -75,64 +78,77 @@ export function Snapshot() {
     const identifier = useContext(SnapshotContext)
     const { payload: proposal } = useProposal(identifier.id)
     const chainId = useChainId(NetworkPluginID.PLUGIN_EVM)
-    const [tabIndex, setTabIndex] = useState(0)
-    const tabs = [
-        <Tab className={classes.tab} key="proposal" label="Proposal" />,
-        <Tab className={classes.tab} key="progress" label="Progress" />,
+    const [currentTab, onChange, tabs] = useTabs('Proposal', 'Progress')
+    const { t } = useI18N()
+    const renderTab = () => {
```
Comment (hanszhoou) [inline packages/mask/src/plugins/Snapshot/SNSAdaptor/Snapshot.tsx:83]: Try `<TabPanel />` ?

**Review by guanbinrui — DISMISSED** [review]

File: packages/mask/src/plugins/Snapshot/SNSAdaptor/Snapshot.tsx [hunk 13]
```diff
@@ -17,55 +20,55 @@ const useStyles = makeStyles()((theme) => {
...
-        subtitle: {
-            fontSize: 12,
-            marginRight: theme.spacing(0.5),
-        },
-        button: {
-            width: '100%',
-            minHeight: 39,
-            margin: `${theme.spacing(1)} auto`,
+        active: {
+            color: '#fff',
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Snapshot/SNSAdaptor/Snapshot.tsx]: Avoid using dead code hex string.

File: packages/mask/src/plugins/Snapshot/SNSAdaptor/Snapshot.tsx [hunk 14]
```diff
@@ -17,55 +20,55 @@ const useStyles = makeStyles()((theme) => {
...
         },
-        choiceButton: {
-            color: theme.palette.mode === 'dark' ? 'white' : 'black',
-            transitionDuration: '0s !important',
-            '&:hover': {
-                border: '2px solid rgb(29, 161, 242) !important',
-                backgroundColor: 'transparent !important',
-            },
+        default: {
+            color: '#fff',
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Snapshot/SNSAdaptor/Snapshot.tsx]: No dead-code hex string.

**Review by guanbinrui — APPROVED** [review]

File: packages/mask/src/plugins/Snapshot/SNSAdaptor/Snapshot.tsx (hunk 8 above)
Comment (guanbinrui) [reply packages/mask/src/plugins/Snapshot/SNSAdaptor/Snapshot.tsx:113]: ?

**Review by UncleBill — APPROVED** [review]

**Pull request merged**